charts/templates/statefulset.yaml

```diff
@@ -66,6 +66,8 @@ spec:
         env: 
         - name: MARKLOGIC_GROUP
           value: {{ .Values.group.name }}
+        - name: XDQP_SSL_ENABLED
```

Comment:
We have used envFrom that imports all the environment variables from ConfigMaps. This should be removed from here.

Reply:
As per the AC, we should have xdqp-ssl-enabled flag in the values file to allow customers to whether or not SSL is enabled for XDQP.

Reply:
That's correct. There are 2 ways to expose the XDQP_SSL_ENABLED env variable, one is like "env:" the other is "envFrom:". We should generally use envFrom instead of env.